src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1412,6 +1412,13 @@ def load_arguments(self, _):
         c.argument('ppg_type', options_list=['--type', '-t'], help="The type of the proximity placement group. Allowed values: Standard.")
         c.argument('tags', tags_type)
 
+    with self.argument_context('ppg create', min_api='2021-11-01') as c:
+        c.argument('zone', nargs='*', help="Specifies the availability zone where virtual machine, virtual machine scale set or availability set associated with the proximity placement group can be created. Accepted values 1,2,3.")
```

Comment:
Could we reuse the `zone_type`
https://github.com/Azure/azure-cli/blob/f7133cb5e2af91ee2cd8673d86885cf292294cd9/src/azure-cli-core/azure/cli/core/commands/parameters.py#L306-L311